src/solve.jl

```diff
@@ -5,6 +5,12 @@ function DiffEqBase.__solve(prob::DiffEqBase.AbstractRODEProblem,
                             kwargs...) where recompile_flag
   integrator = DiffEqBase.__init(prob,alg,timeseries,ts,recompile;kwargs...)
   solve!(integrator)
+  if typeof(prob) <: DiffEqBase.AbstractRODEProblem && typeof(prob.noise) == typeof(integrator.sol.W) && (!haskey(kwargs, :alias_noise) || kwargs[:alias_noise] == true)
```

Comment:
Is the check `... == true` supposed to ensure that a boolean is returned that is only `true` if the keyword argument is set to `true`? That's not sufficient though, e.g., `missing == true` returns `missing`. Only `===` ensures that the result is a boolean.

Otherwise, if you don't care, you could also just check `|| kwargs[:alias_noise]` (but of course that will error if the keyword argument is not a boolean).

Reply:
Thanks, I guess should have used `===`. I think it is better than risking it to error since we are not forcing it to be Boolean. I will change it.